MPCAutofill/MPCAutofill/.env.dist

```diff
@@ -13,6 +13,7 @@ PREPEND_WWW=False
 
 # Postgres Database, create username/pass for security
 DATABASE_ENGINE=django.db.backends.postgresql
+DATABASE_NAME=mpcautofill
```

Comment:
i merged the PR you linked into master, could you back this change out of this PR?

Reply:
Done!